spec/services/feeds/import_spec.rb

```diff
@@ -12,10 +12,11 @@
       end
     end
 
+    # TODO: We could probably improve these tests by parsing against the items in the feed rather than hardcoding
```

Comment:
I was weary of going down a rabbit hole and completely re-writing these tests by parsing the VCR file, so I converted them to a baseline Rspec version, but left a comment here as to how they could be improved!